features/taxation/applying_taxes_based_on_order_item_units_calculation_strategy/applying_correct_taxes_for_item_units_with_discount_for_whole_order_when_tax_rate_included_in_price.feature

```diff
@@ -0,0 +1,50 @@
+@applying_taxes
+Feature: Applying correct taxes for item units with a discount applied for all items in it when tax rates are included in price
+    In order to pay proper amount when buying goods
+    As a Visitor
+    I want to have correct taxes applied when the order is discounted and tax rates are included in products prices
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And default tax zone is "US"
+        And the store uses the "Order item units based" tax calculation strategy configured
+        And the store has included in price "US VAT" tax rate of 23% for "Clothes" within the "US" zone
+        And the store has included in price "Low VAT" tax rate of 10% for "Mugs" within the "US" zone
+        And the store has a product "PHP T-Shirt" priced at "$10.00"
+        And it belongs to "Clothes" tax category
+        And the store has a product "Symfony Mug" priced at "$56.95"
+        And it belongs to "Mugs" tax category
+        And the store has a product "PHP Mug" priced at "$56.90"
+        And it belongs to "Mugs" tax category
+        And there is a promotion "Holiday promotion"
+        And the promotion gives "$10.00" off if order contains a "Symfony Mug" product
+        And there is a promotion "PHP promotion"
+        And the promotion gives "$10.00" off if order contains a "PHP Mug" product
+
+    @ui @api
+    Scenario: Applying correct taxes for a single item unit with order discount
+        When I add product "Symfony Mug" to the cart
+        Then my cart total should be "$46.95"
+        And my included in price taxes should be "$4.27"
+
+    @ui @api
+    Scenario: Applying correct taxes for a single item unit with order discount
+        When I add product "PHP Mug" to the cart
+        Then my cart total should be "$46.90"
+        And my included in price taxes should be "$4.26"
```

Comment:
How is it different from `features/taxation/applying_taxes_based_on_order_item_units_calculation_strategy/applying_correct_taxes_for_item_units_with_discount_for_item_when_tax_rate_included_in_price.feature`? I'm blind or the setup and two first scenarios are the same 😅.

Reply:
The difference is the type of promotion used, in this scenario it is a promotion applied to the whole order and distributed to the units, and in the one you've mentioned, to the unit.